pkg/operator/k8sutil/configmap.go

```diff
@@ -63,7 +63,12 @@ func GetOperatorSetting(context context.Context, clientset kubernetes.Interface,
 
 func GetValue(data map[string]string, settingName, defaultValue string) string {
 	if settingValue, ok := data[settingName]; ok {
-		logger.Infof("%s=%q (configmap)", settingName, settingValue)
+		// Suppress logging of ROOK_WATCH_FOR_NODE_FAILURE setting
+		if settingName == "ROOK_WATCH_FOR_NODE_FAILURE" {
+			logger.Debugf("%s=%q (configmap)", settingName, settingValue)
+		} else {
+			logger.Infof("%s=%q (configmap)", settingName, settingValue)
+		}
 		return settingValue
 	} else if settingValue, ok := os.LookupEnv(settingName); ok {
 		logger.Infof("%s=%q (env var)", settingName, settingValue)
```

Comment:
if someone sets this as ENV we still get this logged as info for every 5 seconds right?

Reply:
right, but we are setting this in configmap only but yes some users may put that in env var so I have added the comment.

Reply:
current approach should fix both your concers @Madhu-1